doc/sphinxext/sphinx_gallery/_static/gallery.css

```diff
@@ -209,7 +209,6 @@ a.sphx-glr-code-links[tooltip]:hover:before{
     display: inline-block;
     text-align: center;
     text-indent: 0;
-    margin-left: -5em;
+    margin-left: 0; /* Has to be zero to avoid overlapping with sidebar */
```

Comment:
Is this change a backport of something in sphinx_gallery, or is it a modification specific for scikit-learn?

I am asking because I am worried that if it is specific to scikit-learn, next time that we upgrade, we will break something.